### PR TITLE
Who made the Tome of Souls work like this

### PR DESCRIPTION
**Pull request by Stagrawl**
And does this fix it?

### CODE REVIEW

File: code/modules/antagonists/roguetown/villain/vampirelord.dm [hunk 1]
```diff
@@ -817,7 +817,7 @@ GLOBAL_LIST_EMPTY(vampire_objects)
 							GLOB.todoverride = null
 							sunstolen = FALSE
 						priority_announce("The Sun is torn from the sky!", "Terrible Omen", 'sound/misc/astratascream.ogg')
-						addomen(OMEN_SUNSTEAL)
+						addomen("sunsteal")
```
Comment (ChungusGamer666) [inline code/modules/antagonists/roguetown/villain/vampirelord.dm:820]: LOUD WRONG BUZZER

**Review by ChungusGamer666 — CHANGES_REQUESTED** [review]
slop 👎

**Comment by Stagrawl** [comment]
I already noticed and fixed it, I don't know why my shit hates me. VSC wants my guts torn out and strewn across the plaza.

File: code/modules/antagonists/roguetown/villain/vampirelord.dm (hunk 1 above)
Comment (Stagrawl) [reply code/modules/antagonists/roguetown/villain/vampirelord.dm]: i alwaeady fixed it!!!!!

**Pull request merged**